src/convertToObject.js

```diff
@@ -12,7 +12,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const propertiesAll = sourceString.split(';');
+  const propertiesTrimmed = propertiesAll.map(property => property.trim());
+  const propertieswithoutEmptyLine = propertiesTrimmed
+    .filter(property => property);
```

Comment:
```suggestion
    .filter(Boolean)
```
це те саме що `.filter(property => Boolean(property))`